app/src/main/java/com/xinto/opencord/ui/screen/ChannelPins.kt

```diff
@@ -144,7 +144,8 @@ private fun ChannelPinsLoaded(
                                 WidgetMessageAuthor(
                                     author = message.author.username,
                                     timestamp = message.formattedTimestamp,
-                                    edited = message.isEdited
+                                    edited = message.isEdited,
+                                    isBot = message.author.bot,
```

Comment:
since `edited` is `isEdited`, maybe `isBot` could be `bot` too? or vice versa, `edited` could be `isEdited`